custom-routes-vis/src/main/webapp/script.js

```diff
@@ -73,6 +77,40 @@ function updateCoordinates(lat, lng, containerId) {
       "Latitude: " + lat + "<br>Longitude: " + lng + "<br>";
 }
 
+/**
+ * Updates the deep linking url to various test apps on the page.
+ * TODO: Update Route Token if we get access to Routes Preferred API.
+ */
+function updateDeepLinkingUrl() {
+  if (document.getElementById('origin-coordinates').innerHTML != '' && 
+      document.getElementById('destination-coordinates').innerHTML != '') {
```

Comment:
Should use `!==` not `!=`.

Reply:
I used `originDestinationMarkers.length == 2 && originDestinationMarkers[0].getVisible() && originDestinationMarkers[1].getVisible()` for a similar condition, but not sure which one is better

Reply:
Actually now that I'm thinking about it, checking they are both visible should be enough(no need to check length) because they are both created to be not visible. And can only become visible after both are created. If you do keep the way you are doing now, you still need ===, not ==.